src/main/groovy/org/reficio/p2/resolver/maven/impl/facade/AetherSonatypeFacade.groovy

```diff
@@ -37,6 +39,7 @@ import org.sonatype.aether.util.graph.PreorderNodeListGenerator
  *         http://www.reficio.org
  * @since 1.1.0
  */
+@edu.umd.cs.findbugs.annotations.SuppressFBWarnings("SE_NO_SERIALVERSIONID")
```

Comment:
Now you have both an import AND a full qualified class.

Reply:
argggh yes thanks :)